.github/CODEOWNERS

```diff
@@ -35,22 +30,22 @@
 /tests/modules/images/webp-uploads                                          @adamsilverstein @felixarntz @mitogh
 /tests/testdata/modules/images/webp-uploads                                 @adamsilverstein @felixarntz @mitogh
 
-# Module: WebP Support Health Check
-/modules/site-health/webp-support                                           @adamsilverstein @kirtangajjar @mitogh
-/tests/modules/site-health/webp-support                                     @adamsilverstein @kirtangajjar @mitogh
-/tests/testdata/modules/site-health/webp-support                            @adamsilverstein @kirtangajjar @mitogh
+# Module: WebP Support
```

Comment:
Please revert the name, as the previous name `WebP Support Health Check` is the correct module name.

Reply:
Module name reverted throughout.